cmd/lotus-bench/import.go

```diff
@@ -229,7 +228,8 @@ var importBenchCmd = &cli.Command{
 		defer cs.Close() //nolint:errcheck
 
 		// TODO: We need to supply the actual beacon after v14
-		stm, err := stmgr.NewStateManager(cs, consensus.NewTipSetExecutor(filcns.RewardFunc), vm.Syscalls(verifier), filcns.DefaultUpgradeSchedule(), nil, metadataDs, index.DummyMsgIndex)
+		stm, err := stmgr.NewStateManager(cs, consensus.NewTipSetExecutor(filcns.RewardFunc), vm.Syscalls(verifier),
+			filcns.DefaultUpgradeSchedule(), nil, metadataDs, nil)
```

Comment:
what's the line breaking rule you're using here? 100 chars? or is it just fitting on your screen? ignore my previous suggestions if there's consistency in your method here